### PR TITLE
code review

### PR DESCRIPTION
**Pull request by smas0900**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# Review-project-from-trials
\ No newline at end of file
+# FCC Test Suite Template
+
```
Comment (smas0900) [inline README.md:2]: Kindly add the Getting Started section and its sub-section as in the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) from Microverse. You can add things like: how to clone the repo, how to open it locally, etc. so the person viewing the project can understand.

File: README.md [hunk 2]
```diff
@@ -1 +1,5 @@
...
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
```
Comment (smas0900) [inline README.md:4]: Kindly add author section so everyone can know who is the author.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
```
Comment (smas0900) [inline index.html:10]: I think it would be better to use semantic tags

File: index.html [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+  <div id="main">
```
Comment (smas0900) [inline index.html:11]: I think it would be better to use class in div instead of id.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+    <h1 id="title">Dr. Norman Borlaug</h1>
+    <p id ="title-description">The man who saved a billion lives</p>
+    <div id="img-div">
+      <img src="image/tribute-page-main-image (1).jpg" alt=" tribut image" id="image">
```
Comment (smas0900) [inline index.html:15]: linters will give error in image name you should rename it

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,75 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+#main{
+    margin: 20px auto;
```
Comment (smas0900) [inline css/style.css:7]: add two spaces of indentation. Do not use the tab key.

**Review by smas0900 — COMMENTED** [review]
Good job so far! clap clap
There are some issues that you still need to work on to go to the next project but you are almost there!


**_Required Changes_** 

Check the comments under the review.

File: README.md (hunk 1 above)
Comment (roshan-bajgain) [reply README.md:2]: - [ ]  Always make sure to include an explanation, a "why", for each requested change, and use polite words. Kindly edit and submit the project.

File: README.md (hunk 2 above)
Comment (roshan-bajgain) [reply README.md:4]: - [ ]  Always make sure to include an explanation, a "why", for each requested change, and use polite words. Kindly edit and submit the project.

File: css/style.css (hunk 6 above)
Comment (roshan-bajgain) [reply css/style.css:7]: - [ ]  Always make sure to include an explanation, a "why", for each requested change, and use polite words. Kindly edit and submit the project.

File: index.html (hunk 3 above)
Comment (roshan-bajgain) [reply index.html:10]: - [ ]  Always make sure to include an explanation, a "why", for each requested change.

File: index.html (hunk 4 above)
Comment (roshan-bajgain) [reply index.html:11]: - [ ]  Always make sure to include an explanation, a "why", for each requested change.

File: index.html (hunk 5 above)
Comment (roshan-bajgain) [reply index.html:15]: - [ ]  Always make sure to use polite words like Kindly, It would be better etc. Kindly edit your comment and submit the project.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>




## Required Changes ♻️ 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
# Hi @smas0900 , 

### Status: Approved ✔️
goodjob


:+1: You have done all the required changes asked by the previous reviewer. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

[Highly Recommended]
- _( optional )_ Please next time you give a review kindly give a reason `"why"` for each comment you add.

_Apart from that you really did a great job_ 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**